### ISSUE TITLE
Set a long numbers or set-add-edit strings like 54:B8:0A:81:B3:18 of menu items inputs.

### ISSUE THREAD

**Issue by HamidSaffari** [littlevgl/lv_apps#7]
Hi, I just want to set long number range like from 0 to 9233304 or even editing it's sign in setting menus and with the Numset it is almost impossible. and one solution is to set each digit individually and add two more buttons Left and Right in addition two +- to do the job and underscored or blinking the editing digit for better visualization.

![xi5Y67u](https://user-images.githubusercontent.com/35362339/79525316-cc94f880-8077-11ea-93af-c8b99209a661.jpg)

Now what do you prefer?

**Comment by HamidSaffari** [comment]
Another solution would be popping-up keyboard that allow you to print it, which is good way also for future because for example you want to set MAC address eg: 54:B8:0A:81:B3:18  or  entering WiFi password so that would be helpful.

**Comment by kisvegabor** [comment]
I don't really understand. So do you want to realize a "number editing" with lvgl similar to the one shown in the image?

**Comment by HamidSaffari** [comment]
No, about the picture is that it editing one digit at the time for it's 4-digit "08:00"and show it by underlined the "8" which is current editing digit.
My question is how it is more efficient to edit long numbers like 8888888 or strings like 54:B8:0A:81:B3:18 etc in your lv_setting method?
becuse your "Numset" with two + and - button is not appropriate for such along number range.
I think about two methods:
1. Is to set each digit individually and add two more buttons Left and Right in addition two +- in 
   your "Numset" to go to next/previous digit.
2. Is popping-up keyboard that allows you to print the number or strings, which is good way also for 
   future because for example you want to set MAC address eg: 54:B8:0A:81:B3:18 or entering WiFi 
   password so that would be helpful.

**Comment by kisvegabor** [comment]
I see. 
I think adding a general text input would be a more reasonable direction to take because it could be used for other purposes too.

Or the spinbox object for numbers only. 
https://docs.littlevgl.com/en/html/object-types/spinbox.html

Unfortunately, at this time I'm busy with developing v7 so I can not add it now. :(
